src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -296,6 +362,216 @@ const formConfig = {
             }
           }
         },
+        // TODO: Should this be privateMedicalRecordsRelease or privateRecordsRelease? 
```

Comment:
:thinking: I think either way is fine. If you want to shorten it, it's still clear since there are no _other_ private records which we have to upload (to my knowledge).

I'm good with whatever you want to do here. (Within reason, of course :stuck_out_tongue_winking_eye:)

Reply:
OK, I'll leave this as is for consistency sake.